utils/dev/build-lockbook-debian/release_linux_cli_debian.sh

```diff
@@ -1,105 +0,0 @@
-#!/bin/bash
-
-set -ae
```

Comment:
I believe this bit is not yet in the releaser, probably the only thing of value here is. I'm fine with still deleting it and just referring to this comment. 

```
dch -v $new_version "Automatic version bump."

echo "Setting up clean environment"
debuild -- clean

echo "Compiling package"
debuild --no-lintian
```